src/Simplex/Chat/Store/Messages.hs

```diff
@@ -1733,11 +1734,10 @@ deleteChatItemVersions_ :: DB.Connection -> ChatItemId -> IO ()
 deleteChatItemVersions_ db itemId =
   DB.execute db "DELETE FROM chat_item_versions WHERE chat_item_id = ?" (Only itemId)
 
-markDirectChatItemDeleted :: DB.Connection -> User -> Contact -> ChatItem 'CTDirect d -> MessageId -> UTCTime -> IO (ChatItem 'CTDirect d)
-markDirectChatItemDeleted db User {userId} Contact {contactId} ci@ChatItem {meta} msgId deletedTs = do
+markDirectChatItemDeleted :: DB.Connection -> User -> Contact -> UTCTime -> CChatItem 'CTDirect -> IO (CChatItem 'CTDirect, CChatItem 'CTDirect)
+markDirectChatItemDeleted db User {userId} Contact {contactId} deletedTs item@(CChatItem msgDir ci@ChatItem {meta}) = do
   currentTs <- liftIO getCurrentTime
-  let itemId = chatItemId' ci
-  insertChatItemMessage_ db itemId msgId currentTs
```

Comment:
wasn't it needed?

Reply:
it wasn't necessary - just some legacy bookkeeping but it more gets in the way than helps here. worst thing this can affect is this message record won't be cascade deleted if chat item record is later deleted (here it's not deleted but only updated), but messages are periodically deleted anyway